extensions/dacpac/src/wizard/dataTierApplicationWizard.ts

```diff
@@ -390,4 +528,20 @@ export class DataTierApplicationWizard {
 		}
 		return this.dacfxService;
 	}
+
+	public getDacServiceArgsAsProps(service: mssql.IDacFxService, database: string, filePath: string, ownerUri: string): { [k: string]: string } {
+		return {
+			isServiceExist: (!utils.isNullOrUndefined(service)).toString(),
```

Comment:
Instead of making these helper functions just do something like this

`isServiceExist: (!!service).toString()` (false if undefined/null, true otherwise)
`isDatabaseExists: (!!database).toString()` (false if empty/undefined/null, true otherwise)